src/datepicker/datepicker-input.spec.ts

```diff
@@ -0,0 +1,220 @@
+import {TestBed, ComponentFixture, fakeAsync, tick} from '@angular/core/testing';
+import {By} from '@angular/platform-browser';
+import {createGenericTestComponent} from '../test/common';
+
+import {Component} from '@angular/core';
+import {FormsModule} from '@angular/forms';
+
+import {NgbDatepickerModule} from './datepicker.module';
+import {NgbInputDatepicker} from './datepicker-input';
```

Comment:
I'm fine with this - the regular datepicker is `NgbDatepicker`, we namespace all of these for clarity that these are third party components & it creates a nice user cognition that they should be prefixing the ngb directives/components.